README.md

```diff
@@ -38,6 +38,21 @@ and `mocker.patch('target', lambda *args: 7)` is an error
 
 * `PT009 use a regular assert instead of unittest-style '{assertion}'`
 
+* `PT010 set the match parameter in pytest.raises`  
+e.g. `pytest.raises(ValueError, match='exception text')` is OK,
+and `pytest.raises(ValueError)` is an error.
+    * Since this is a fairly disruptive rule, it is by default off. 
+      You can turn it on by specifying in your `setup.cfg` file 
```

Comment:
`setup.cfg` is not the only way to configure Flake8, so I would phrase this differently:
```suggestion
      You can turn it on by specifying in your configuration (e.g. `setup.cfg`) 
```